manager/controllers/app/m4dapplication_controller.go

```diff
@@ -115,6 +115,9 @@ func (r *M4DApplicationReconciler) Reconcile(req ctrl.Request) (ctrl.Result, err
 	if hasError(applicationContext) {
 		log.Info("Reconciled with errors: " + getErrorMessages(applicationContext))
 	}
+	if !applicationContext.Status.Ready {
+		return ctrl.Result{Requeue: true}, nil
```

Comment:
would it make more sense to use RequeueAfter?

Reply:
OK, changed to requeue after 10 seconds